src/test/java/de/tum/in/www1/artemis/util/DatabaseUtilService.java

```diff
@@ -3871,6 +4589,7 @@ public ExampleSubmission generateExampleSubmission(String modelOrText, Exercise
     }
 
     /**
+     * Moved to {@link QuizExerciseUtilService#generateSubmittedAnswerFor(QuizQuestion, boolean)}
```

Comment:
this and many other methods below related to creating quizes should be moved into the QuizFactory instead

Reply:
Discussed: Done in FollowUps